specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-09-02-preview/managedClusters.json

```diff
@@ -3562,6 +3585,48 @@
       },
       "description": "Contains the IPTag associated with the object."
     },
+    "PortRange": {
+      "type": "object",
+      "description": "The port range.",
+      "properties": {
+        "portStart": {
+          "type": "integer",
+          "format": "int32",
+          "minimum": 1,
+          "maximum": 65535,
+          "description": "The minimum port that is included in the range. It should be ranged from 1 to 65535, and be less than or equal to `portEnd`."
```

Comment:
minor: remove the backticks here and below - they don't render well in most places that render this text.

Reply:
fixed